selfdrive/ui/mici/onroad/driver_camera_dialog.py

```diff
@@ -60,6 +60,9 @@ def hide_event(self):
     ui_state.params.put_bool("IsDriverViewEnabled", False)
     device.reset_interactive_timeout()
 
+  def close_dialog(self):
+    gui_app.set_modal_overlay(None)
```

Comment:
Why do we have to make everything methods?

Reply:
Because WeakSet only supports weakly-referenced bound methods, adding lambdas or plain functions would complicate the design. Supporting bound-method callbacks is sufficient here, and we can use a separate class when we need more flexible callback types for a single-callback case.